src/app/not-found.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button } from '@/components/Button'
+
+export default function NotFound() {
+  return (
+    <div className="xs:items-center xs:text-center m-auto flex max-w-xs flex-col gap-6">
+      <div className="h-[104px]"></div>
+      <span className="text-8xl">404</span>
+      <h2 className="text-3xl">Page Not Found</h2>
```

Comment:
Perhaps we should talk to @filipagr about taking these font sizes down a notch on `xs`

Reply:
Hey! Already discussed this with Barbara and if we're taking the Page not Found font size down a notch on mobile, then we should also do it on the other screens as well, because we're using the same font sizes. The 404 font size was already taken down to 7xl, instead of 8xl. The same on the 500 error page.

Reply:
@filipagr We can look into that another time, thank you.